src/components/features/SignSection.vue

```diff
@@ -8,8 +8,8 @@ export default defineComponent({
 
 <template>
   <div class="sign-section">
-    <AppButton to="/" size="medium" colorScheme="no-border">Register</AppButton>
-    <AppButton size="medium" colorScheme="main">Log in</AppButton>
+    <AppButton to="/" variant="internal-link" size="medium" colorScheme="no-border">Register</AppButton>
```

Comment:
I highly suggest to use named routes instead of the strings (and make the component able to accept both strings and objects on this prop).
This way we don't need to worry if paths change in the future 😄 

```suggestion
    <AppButton :to="{ name: 'Home' }" variant="internal-link" size="medium" colorScheme="no-border">Register</AppButton>
```

Reply:
I actually tried to apply this good practice, but I failed and only now I know why -- no object type accepted as a prop by the AppButton! Now everything works seamlessly! Thanks a lot! 🚀 😃